cli/src/pcluster/cli/commands/configure/easyconfig.py

```diff
@@ -108,7 +108,7 @@ def _get_subnets(conn, vpc_id):
 
 def configure(args):  # noqa: C901
 
-    config_file_path = args.config if args.config else default_config_file_path()
+    config_file_path = args.config
```

Comment:
we should remove the default_config_file_path from the import and from the utils file. the default_config_file_path shouldn't be used anywhere.

Reply:
Done. The method is removed from utils